qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -40,10 +42,14 @@ def __init__(self, parent, font=None):
         if not font:
             font = text_font()
 
+        completion_enabled = True
+        if CONF.has(ENABLE_COMPLETION_KEY):
+            completion_enabled = CONF.get(ENABLE_COMPLETION_KEY)
```

Comment:
Would it be more concise to do this instead
```suggestion
        completion_enabled = CONF.get(ENABLE_COMPLETION_KEY, True)
```

Reply:
I am not sure I understand this proposed change. First, it is not possible to run get with the second being `True`, as the sanity check inside enforces this parameter to be a string. Second, if the change is made so the sanity check passes, and the second is now "True", that creates a setting with the full name (in this case): `Editors/completion_enabled/True`, which does not exist and the configuration lookup fails. Does this work on your side? If so, could you elaborate?

Reply:
Ignore me, I think I assumed `CONF` was a dictionary for some reason!